.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,76 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master, dev-build ]
+
+env:
+  ORG: timescale # timescaledev
+  
+jobs:
+
+  timescaledb:
+
+    name: PG${{ matrix.pg }}${{ matrix.oss }}
+    runs-on: ubuntu-latest
+    strategy: 
+      matrix:
+        pg: [ 9.6, 10, 11, 12 ]
```

Comment:
9.6 and 10 are no longer supported

Reply:
@svenklemm There are two reasons for covering all four PGs:
- 1.7.1 supports all for PG versions and this PR is applicable to 1.7.1 and actually it is good to rebuild multi-arch images, since I wasn't able to build all targeted architectures.
- We might release 1.7.2 and it will be easier to have the reference version.

9.6 and 10 will be dropped together with 2.0 beta release.

Reply:
No, it is not possible. Neither sequences are supported in environment variables, nor expressions and thus environment variables can be used in `matrix`.